package.js

```diff
@@ -20,7 +20,9 @@ Package.onUse(function (api) {
     'check'
   ], both)
 
-  api.use(['blaze@2.5.0'], 'client', { weak: true })
+  api.use('zodern:types')
+
+  api.use(['blaze@2.6.2'], 'client', { weak: true })
```

Comment:
Why Blaze 2.6.2 and not 2.7?

Reply:
It is a weak dependency, so I'm going it incrementally. But that is something that can be bumped further for the release if you want.